src/blockchain-runtime-interface.md

```diff
@@ -59,6 +59,11 @@ These functions can return the following errors:
 - `InsufficientRegistrations`; too few mixnodes were registered for the session (`num`, less than
   the minimum `min`). The mixnet is not operational in such sessions, although nodes should still
   handle traffic for the previous session in the first 3 phases.
+
+  // TODO this implies that we need to manually register nodes: should be good to define how it is
+  // done (offchain worker putting a extrinsic): this needs to be known by implementors (even if offchain
+  should be called by those correctly there may be some quirks).
```

Comment:
What bit of this do you think implies we need to manually register nodes?

As you probably saw on the other PR, Basti has asked me to change how the registration stuff is done... I'll push that update and a spec update soon. As the registrations will be more actively driven by the node, I'll need to say a bit more in this spec...